lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -375,12 +377,12 @@ global proc string ClearAllUSDReferencesConfirm()
 ClearAllUSDReferencesConfirm();
 )";
 
-class AddReferenceUndoableCommand : public Ufe::UndoableCommand
+class AddUsdReferenceUndoableCommand : public Ufe::UndoableCommand
```

Comment:
Renamed for clarity: existing code creates a USD reference, not a Maya reference prim.